setup.py

```diff
@@ -32,7 +32,14 @@
 
     keywords='Messages',
     packages=find_packages(exclude=['contrib', 'docs', 'tests']),
-    install_requires=[],
+    install_requires=[
+        'amqp==2.2.2',
+        'billiard==3.5.0.3',
+        'celery==4.1.0',
+        'kombu==4.1.0',
+        'pytz==2017.3',
+        'vine==1.1.4'
+    ],
```

Comment:
This list comes from the requirements.txt file in this repo? because the only real dependency here is `Celery` the others packages are celery's dependencies right?

Reply:
I agree with all, only one concern: "... allow to Celery manage his own dependencies..." means that in our setup and requirements.txt we only have to specify "celery==4.x.x", right?